lib/tasks/plugin_assets.rake

```diff
@@ -43,17 +45,28 @@ task 'plugin:assets:precompile', [:plugin] => [:environment] do |t, args|
     class PluginWebpackTask
       attr_accessor :plugin
 
-      def initialize(plugin_id)
-        @plugin = Foreman::Plugin.find(plugin_id) or raise("Unable to find registered plugin #{plugin_id}")
+      def initialize(plugin_id, plugin: nil)
+        @plugin = plugin || Foreman::Plugin.find(plugin_id) or raise("Unable to find registered plugin #{plugin_id}")
       end
 
       def compile
         return unless File.exist?("#{@plugin.path}/webpack")
         return unless File.exist?("#{@plugin.path}/package.json")
         ENV["NODE_ENV"] ||= 'production'
-        webpack_bin = ::Rails.root.join('node_modules/webpack/bin/webpack.js')
-        config_file = ::Rails.root.join(::Rails.configuration.webpack.config_file)
-        sh "node --max_old_space_size=2048 #{webpack_bin} --config #{config_file} --bail --env.pluginName=#{@plugin.id}"
+        # TODO: delete this as all plugins get compiled in the regura
```

Comment:
How would we resolve this?

Reply:
Updated the code to continue working as before and only compiling one plugin for this task